neqo-transport/src/connection/params.rs

```diff
@@ -81,6 +82,7 @@ impl Default for ConnectionParameters {
             ack_ratio: DEFAULT_ACK_RATIO,
             idle_timeout: DEFAULT_IDLE_TIMEOUT,
             preferred_address: PreferredAddressConfig::Default,
+            max_datagram_frame_size: 0,
```

Comment:
I think that we want the default for this to be enabled.  I realize that you don't want this in the HTTP/3 part just yet (I haven't worked out why yet), but I think - long term - we will want to negotiate this feature for those connections too.

Reply:
I will leave the feature to be disabled by default for now. 
I remove the assertion in HTTP/3 code.